Modules/Invoices/Peppol/Services/PeppolTransformerService.php

```
@@ -0,0 +1,165 @@
+            'payment_terms' => $this->transformPaymentTerms($invoice),
+            
+            // Metadata
+            'format' => $format,
+            'invoice_id' => $invoice->id,
+        ];
+    }
+
+    /**
+     * Get invoice type code (380 for standard invoice, 381 for credit note)
+     */
+    protected function getInvoiceTypeCode(Invoice $invoice): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$invoice'. (undefined)

(UnusedFormalParameter)